resources/scss/product.scss

```diff
@@ -0,0 +1,200 @@
+@import "mixins.scss";
+@import "variables.scss";
+.products-container {
+  padding: 0 40px;
+  display: flex;
+}
+
+.prodlist-container {
+  width: 85%;
+}
+
+.product-item {
+  width: 20%;
+  float: left;
+  margin: 15px;
+  height: 475px;
+  &::after {
+    content: "";
+    display: block;
+    border: 1px dotted #1d151533;
+  }
+  h4 {

```

Comment:
element level styling should only be part of your global and reset css, rest use classes